File: scripts/maintenance/inspect_json.py

import json

try:
    with open('generation_results.json', 'r') as f:
        data = json.load(f)
        
    if "energy_summary" in data:
        print("Keys in energy_summary:", list(data["energy_summary"].keys()))
        # Print a sample entry
        sample_key = list(data["energy_summary"].keys())[0] if data["energy_summary"] else None
        if sample_key:
            print(f"Sample data for {sample_key}:", data["energy_summary"][sample_key])
            
    else:
        print("energy_summary key NOT found in JSON")
        
    if "countries" in data:
        print("Keys in countries:", list(data["countries"].keys()))

except Exception as e:
    print(f"Error: {e}")

File: scripts/maintenance/inspect_json.js

const fs = require('fs');

try {
    const raw = fs.readFileSync('generation_results.json');
    const data = JSON.parse(raw);

    if (data.energy_summary) {
        console.log("Keys in energy_summary:", Object.keys(data.energy_summary));
        const sampleKey = Object.keys(data.energy_summary)[0];
        console.log(`Sample data for ${sampleKey}:`, data.energy_summary[sampleKey]);
    } else {
        console.log("energy_summary key NOT found");
    }

    if (data.countries) {
        console.log("Keys in countries:", Object.keys(data.countries));
    }

} catch (err) {
    console.error(err);
}
